### ISSUE TITLE
Bei Wiedergabe wird in 90% der Fälle an's Ende der Aufnahme gesprungen

### ISSUE THREAD

**Issue by Dieter59** [betonme/e2openplugin-EnhancedMovieCenter#287]
01
wenn ich Aufnahmen aus EMC heraus starte kommt es extrem oft (aber halt nicht immer) vor, dass die Wiedergabe ganz an's Ende der Aufnahme springt. Starte ich die Aufnahmen mit der Movielist, passiert das nicht. Hat irgendjemand eine Ahnung woran das liegen könnte. Ich hatte die Einstellungen von EMC im Verdacht, aber da wüsste ich nicht, was ich anders machen könnte.
![47CB6FBB-6768-4C57-9C7D-F5292444B753](https://user-images.githubusercontent.com/32402895/145713754-dc64f6cb-6c18-4a11-8589-e50592fd56fd.jpeg)

**Comment by pjsharp** [comment]
Besteht das Problem immer noch? Wie kann man sich das genau vorstellen? Passiert es gleich nach dem Start dass die Wiedergabe ans Ende springt oder erst beim Vorspulen?

**Comment by Dieter59** [comment]
Ja, das Problem besteht nach wie vor. Es passiert sofort heim Start des Films. Ich konnte es einschränken: es passiert fast immer bei Aufnahmen, die mit VPS von den ÖR aufgenommen wurden, da es da quasi keinen Vorlauf gibt. Ich könnte mir vorstellen, dass es zwar einen Startmarker gibt, der dann aber sofort am Anfang steht und übersprungen wird. Interessanterweise passiert das dann aber beim zweiten Start nicht mehr.

**Comment by maxmadr** [comment]
Genau das Gleiche hier. Wenn man versucht einen Film abzuspielen, den man noch nie angesehen hat, dann zeigt die Filmliste das kurz an springt dann aber sofort zum Ende des Film und zeigt im Display den Film als gesehen an. Dann zählt der Film sogar noch über das Ende des Films hinaus. Ich muss mir dann immer so behelfen ein Stück zurückzuspulen, dann den Film stoppen und ihn nochmal vom nun neu gesetzten Marken zu starten und dann nochmals ganz zurückzuspulen. Das funktioniert manchmal. Echt ätzend.
